client/src/components/Auth/PrivateRoute/__tests__/PrivateRoute.test.jsx

```diff
@@ -0,0 +1,57 @@
+import React from "react";
+import { MemoryRouter } from "react-router";
+import { mount } from "enzyme";
+import { findByTestAttr } from "../../../../test/testUtils";
+import { GenericComponent } from "../../../../test/__mocks__/GenericComponent";
+import PrivateRoute from "../PrivateRoute";
+import thunk from "redux-thunk";
+import configureStore from "redux-mock-store";
+
+const mockStore = configureStore([thunk]);
+
+const loggedInState = {
+  auth: {
+    isAuthenticated: true
+  }
+};
+
+const loggedOutState = {
+  auth: {
+    isAuthenticated: false
+  }
+};
+
+describe("PrivateRoute rendering", () => {
+  test("it should render a component if user is logged in", () => {
+    const store = mockStore(loggedInState);
+    let wrapper = mount(
+      <MemoryRouter initialEntries={["/dashboard"]} initialIndex={0}>
+        <PrivateRoute
+          path="/dashboard"
+          component={GenericComponent}
+          store={store}
+        />
+      </MemoryRouter>
+    );
+    const generic = findByTestAttr(wrapper, "generic");
```

Comment:
In this case since you know exactly what the component is (and you're importing it), you can probably just say `expect(wrapper.find(GenericComponent).exists()).toBe(true)` instead of using this helper function.

Reply:
yup. that makes sense.